# Positions of the orange objects

Header header

Targets targets